puppet/modules/web/manifests/vhost/stagingyum.pp

```diff
@@ -6,6 +6,7 @@
   String $user = 'yumrepostage',
   Stdlib::Absolutepath $home = "/home/${user}",
   Integer[0] $rsync_max_connections = 5,
+  Array[String[1]] $usernames = ['ehelms', 'evgeni', 'ekohl'],
```

Comment:
Something tells me this list should be configured somewhere else, but I guess it's good enough here for starting and testing?